docs/Model_Assumptions.rst

```diff
@@ -287,6 +287,89 @@ For example, the `degree of sector coupling` will reach its maximum, when the sy
 
+CO2 Emission factors
+----------------------------
+In order to optimise the energy system with minimum CO2 emissions, it is important to calculate CO2 emission per unit of fuel consumption.
+
+Following emission factors for energy carriers are defined:
+
+.. _table_CO2_emissions_energyCarriers:
+
+.. list-table:: CO2 Emission factors: Kg of CO2 equivalent per unit of fuel consumption
+   :widths: 50 25 25 25
+   :header-rows: 1
+
+   * - Energy carrier
+     - Unit
+     - Value
+     - Source
+   * - Diesel
+     - kgCO2eq/litre
+     - 2.68
+     - [`4  <https://www.ipcc-nggip.iges.or.jp/public/2006gl/vol2.html>`__]
+   * - Gasoline
+     - kgCO2eq/litre
+     - 2.29
+     - [`4  <https://www.ipcc-nggip.iges.or.jp/public/2006gl/vol2.html>`__]
+   * - Biodiesel
+     - kgCO2eq/litre
+     - 0.03178
+     - [`4  <https://www.ipcc-nggip.iges.or.jp/public/2006gl/vol2.html>`__]
+   * - Bioethanol
+     - kgCO2eq/litre
+     - 0.00855
+     - [`4  <https://www.ipcc-nggip.iges.or.jp/public/2006gl/vol2.html>`__]
+   * - LPG
+     - kgCO2eq/litre
+     - 1.611
+     - [`4  <https://www.ipcc-nggip.iges.or.jp/public/2006gl/vol2.html>`__]
+   * - Biogas
+     - kgCO2eq/m3
+     - 0
+     - [`5 <https://doi.org/10.1080/10934529.2018.1459076>`__]
+   * - Natural gas
+     - kgCO2eq/m3
+     - 1.88
+     - [`4  <https://www.ipcc-nggip.iges.or.jp/public/2006gl/vol2.html>`__]
+
+
+The CO2 emission of biogas is considered neutral since CO2 is considered as biogenic.[`5 <https://doi.org/10.1080/10934529.2018.1459076>`__]
+
+CO2 emissions for Germany and the four pilot sites (Norway, Spain, Romania, India):
+
+.. _table_CO2_emissions_countries:
+
+.. list-table:: CO2 Emission factors: grams of CO2 equivalent per kWh of electricity consumption
+   :widths: 50 25 25 25
+   :header-rows: 1
+
+   * - Country
+     - Value
+     - Unit
+     - Source
+   * - Germany
+     - 397
+     - kgCO2eq/kWh
+     - [`6 <https://www.agora-energiewende.de/fileadmin2/Projekte/2019/Jahresauswertung_EU_2019/172_A-EW_EU-Annual-Report-2019_Web.pdf>`__]
+   * - Norway
+     - 14.4
+     - kgCO2eq/kWh
```

Comment:
In the source I found the unit `gCO2/kWh` so here the value should be 0.0144, I think the same accounts for the other countries.

Reply:
Yes. This was a mistake in writing units. I have changed it to `gCO2 eq/kWh`.